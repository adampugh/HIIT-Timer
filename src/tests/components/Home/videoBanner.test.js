import React from "react";
import { shallow } from "enzyme"; 

import VideoBanner from "../../../components/Home/videoBanner";

test("should render videoBanner correctly", () => {
    const wrapper = shallow(<VideoBanner />);
    expect(wrapper).toMatchSnapshot();
});

// test for modal opening function?
// handleOpenModal={false}