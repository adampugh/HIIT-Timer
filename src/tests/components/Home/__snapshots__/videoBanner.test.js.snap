// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render videoBanner correctly 1`] = `
<header
  id="header"
>
  <div
    className="video__container"
  >
    <div
      className="video__overlay"
    >
      <h1>
        HIIT Timer
      </h1>
      <p>
        Create awesome workouts and get in shape!
      </p>
      <button
        className="btn"
      >
        Log In
      </button>
      <socialLinks />
    </div>
    <video
      autoPlay={true}
      loop={true}
      muted={true}
      playsInline={true}
      poster="Workout.jpg"
    >
      <source
        src="Workout.mp4"
        type="video/mp4"
      />
      <source
        src="Workout.webm"
        type="video/webm"
      />
    </video>
  </div>
</header>
`;
